crypto_kem/mceliece348864/avx/int32_sort.h

```diff
@@ -0,0 +1,9 @@
+#ifndef int32_sort_h
```

Comment:
why do  some of the header guards have namespacing and others don't?

Reply:
Consequence of this being a manual effort – we only really require this for ``api.h`` files. I'll see if I can craft a regex.

Reply:
```sh
 perl -p -e 's/^(#(ifndef|define)) +(MC_)?(([a-zA-Z0-9_]|_)+_[hH])$/\1 MC_\U\4/' -i $(find schemes -iname '*.h')
```